cli/format/format_extended.go

```diff
@@ -89,6 +89,23 @@ func (formatter *ExtendedFormatter) FormatConfig(status *models.AlertmanagerStat
 	return nil
 }
 
+// FormatClusterStatus formats the cluster status with peers into a readable string
+func (formatter *ExtendedFormatter) FormatClusterStatus(status *models.ClusterStatus) error {
+	w := tabwriter.NewWriter(formatter.writer, 0, 0, 2, ' ', 0)
+	fmt.Fprintf(w, "Cluster Status:\t%s\n\n", *status.Status)
```

Comment:
I would add the node's name (e.g. `status.Name`).

Reply:
I added the node name into the simple format as well